### PR TITLE
created tile component

### PR DESCRIPTION
**Pull request by talk2coded**

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import Slider from "react-input-slider";
 import fetch from "isomorphic-unfetch";
 import Error from "../Components/Error";
 import DrumPad from "../Components/DrumPad";
+import DrumLoopTile from "../Components/DrumLoopTile/index";
```
Comment (rgavinc) [inline src/pages/index.js]: - [ ] For now until we have our designs we are not going have the DrumLoopTile appear on index.js. Can you remove it from here for now?

File: src/Components/DrumLoopTile/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styled from "styled-components";
+
+const TitleContainer = styled.div`
+  border: 1px solid black;
+  width: 200px;
```
[the remark this answers is not in the document]
Comment (talk2coded) [reply src/Components/DrumLoopTile/index.js]: it has been changed

File: src/pages/index.js (hunk 1 above)
Comment (talk2coded) [reply src/pages/index.js]: okay


> Hi @talk2coded,
> 
> Awesome work!! :) Thanks for the contribution. I just ask that you fix the merge conflict.
> 
> Also I strongly encourage you to write a test for this component. To start you can write a test that will just assert that this component renders properly with default props. If you need help with it please let me know :)

how do i do that ?

Comment (rvvergara) [reply src/pages/index.js]: Great! Please check the other components (in `src/components` folder) and look for files called `index.test.js` for examples on how to do this. Have you used `jest` and/or `enzyme` before? If not please refer to this [resource](https://www.freecodecamp.org/news/testing-react-hooks/) as a guide

Please try to do this and let me know how everything goes.

Comment (talk2coded) [reply src/pages/index.js]: i have done it

File: src/Components/DrumLoopTile/index.test.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import ReactDOM from "react-dom";
+import LoopTile from "./index";
+
+import Enzyme, { shallow, render, mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import Adapter from "enzyme-adapter-react-16";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+/*it("renders correctly enzyme", () => {
```
Comment (rvvergara) [inline src/Components/DrumLoopTile/index.test.js]: Is it ok if we just remove these commented code? :)

Comment (talk2coded) [reply src/Components/DrumLoopTile/index.test.js]: There's no problem, It has been done

**Pull request merged**